lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -594,18 +594,21 @@ class FlDotData {
   /// override [checkToShowDot] to handle it in your way.
   const FlDotData({
     this.show = true,
-    this.dotColor = Colors.blue,
     this.dotSize = 4.0,
     this.checkToShowDot = showAllDots,
+    this.strokeWidth,
+    this.getStrokeColor,
+    this.getDotColor = _defaultGetDotColor,
   });
 
   /// Lerps a [FlDotData] based on [t] value, check [Tween.lerp].
   static FlDotData lerp(FlDotData a, FlDotData b, double t) {
     return FlDotData(
       show: b.show,
       checkToShowDot: b.checkToShowDot,
-      dotColor: Color.lerp(a.dotColor, b.dotColor, t),
       dotSize: lerpDouble(a.dotSize, b.dotSize, t),
+      strokeWidth: lerpDouble(a.strokeWidth, b.strokeWidth, t),
+      getDotColor: b.getDotColor,
```

Comment:
add `getStrokeColor` in the lerp function.

Reply:
fixed